pjmedia/include/pjmedia/endpoint.h

```diff
@@ -72,6 +72,16 @@ typedef struct pjmedia_endpt_create_sdp_param
      */
     pjmedia_dir dir;
 
+    /**
+     * Specifies whether redundancy should be offered in the SDP.
+     * Value is integer, indicating redundancy levels (0 means disabled/
+     * no redundancy).
+     * Currently it only applies to text media SDP.
+     *
+     * Default: PJMEDIA_DEFAULT_REDUNDANCY_LEVEL
+     */
+    int         red_level;
```

Comment:
Perhaps add some hints/links about redundancy docs here or somewhere else (if not yet), e.g: what it is, what level represents & the effect/risk, valid level values (if relevant), and how it is negotiated.